### PR TITLE
Fix typo in superposition workbook part2

### PR DESCRIPTION
**Pull request by delight1019**
Hello.

I fixed a typo in superpositio workbook part2, which misses close parenthesis.

### CODE REVIEW

File: Superposition/Workbook_Superposition_Part2.ipynb [hunk 1]
```diff
@@ -1392,7 +1392,7 @@
     "3. If we take this one step further, to $N = 4$, we'll see that the same recursive logic can be applied to the larger and larger sizes of the problem. Indeed,\n",
     "\n",
     "$$|W_4\\rangle = \\frac{1}{2}\\big(|1000\\rangle + |0100\\rangle + |0010\\rangle + |0001\\rangle\\big) = \\\\\n",
-    "= \\frac{1}{\\sqrt2} \\big(\\frac{1}{\\sqrt2}(|10\\rangle + |01\\rangle) \\otimes |00\\rangle + |00\\rangle \\otimes \\frac{1}{\\sqrt2}(|10\\rangle + |01\\rangle) = \\\\\n",
+    "= \\frac{1}{\\sqrt2} \\big(\\frac{1}{\\sqrt2}(|10\\rangle + |01\\rangle) \\otimes |00\\rangle + |00\\rangle \\otimes \\frac{1}{\\sqrt2}(|10\\rangle + |01\\rangle)) = \\\\\n",
```
Comment (tcNickolas) [inline Superposition/Workbook_Superposition_Part2.ipynb:1395]:
```suggestion
    "= \\frac{1}{\\sqrt2} \\big(\\frac{1}{\\sqrt2}(|10\\rangle + |01\\rangle) \\otimes |00\\rangle + |00\\rangle \\otimes \\frac{1}{\\sqrt2}(|10\\rangle + |01\\rangle) \\big) = \\\\\n",
```

**Review by tcNickolas — CHANGES_REQUESTED** [review]
Nice catch! 
Just one minor suggestion: the closing bracket need to match the opening `\big(`, so it needs to be marked as `\big)` as well. I left this as a suggestion so that it's easier to apply.

Thank you!

**Review by tcNickolas — APPROVED** [review]
I remembered that I can actually apply my own suggestions, so I went ahead and did just that :-)

**Comment by delight1019** [comment]
@tcNickolas Thanks for suggestion! I could not check your comment since I've taken my vacation on this weekend. I'll remember it for next time :)

**Pull request merged**